Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM wisvch/debian:stretch
+MAINTAINER Mark Janssen <mark@praseodym.net>
+
+RUN apt-get update && apt-get install -y nodejs nodejs-legacy && rm -rf /var/lib/apt/lists/*
+COPY florriebot.tar.gz /tmp
```

Comment:
The other option would be to do an `npm install` in the container, but this requires many dependencies that make the image a lot bigger. I could include some instructions in the README file though.